website/docs/features/templates.md

```diff
@@ -6,10 +6,11 @@ Taqueria plugins are able to define templates that can be used to generate proje
 
 ## Available Templates
 
-| Template           | < template >         | Plugin                      | Command                       | Description                                                                                                        |
-|--------------------|----------------------|------------------------------|------------------------------|--------------------------------------------------------------------------------------------------------------------|
-| Ligo contract      | `contract`           | `@taqueria\plugin-ligo`      | `taq create contract <path>` | Instantiates a new Ligo contract at the provided `<path>` and registers the contract in the contract registry      |
-| Archetype contract | `archetypeContract`  | `@taqueria\plugin-archetype` | `taq create contract <path>` | Instantiates a new Archetype contract at the provided `<path>` and registers the contract in the contract registry |
+| Template                              | < template >        | Plugin                       | Command                                   | Description                                                                                                        |
```

Comment:
Btw were you able to build the website?

Reply:
Yes
<img width="984" alt="image" src="https://user-images.githubusercontent.com/10706087/189185373-c97e931d-fc73-4584-9f51-bcfd345c6493.png">